src/main/webapp/WEB-INF/tags/shared/commentsPanel.tag

```diff
@@ -21,65 +21,89 @@
 </c:choose>
 <div class="panel panel-primary">
     <div class="panel-heading">
```

Comment:
With regards to the collapsing of the panel, perhaps using the data-toggle="collapse" attribute on the panel-heading would be sufficient, instead of using bindCollapseEvents(). You could move the toggleChevron() (as well as setChevronToUp() and setChevronToDown()) method from feedbackResponseComments.js to common.js, and use it to perform the toggling of the chevron.